storage/tests/unit/test_bucket.py

```diff
@@ -14,8 +14,37 @@
 
 import datetime
 import unittest
-
+from six.moves import http_client
+import requests
 import mock
+import json
+
+
+def _make_response_error(
+    status=http_client.INTERNAL_SERVER_ERROR, content=b"", headers={}
+):
+    response = requests.Response()
+    response.status_code = status
+    response._content = content
+    response.headers = headers
+    response.request = requests.Request()
+    return response
+
+
+def _make_requests_session(responses):
+    session = mock.create_autospec(requests.Session, instance=True)
+    session.request.side_effect = responses
+    return session
+
+
```

Comment:
Keep single empty line between methods, unless entire file uses double-lines.

Reply:
That is PEP8 Style guide for python code, so in that if methods are not in class or in main need to keep two blank line between class or methods but if methods in the class than use single blank line between them, if you are using Pycharm IDEE for python than you found instruction about PEP8 when you don't follow that guide line